src/main/java/com/syncstream_dev/syncstream/model/session/PrivateSession.java

```diff
@@ -0,0 +1,24 @@
+package com.syncstream_dev.syncstream.model.session;
+
+import com.syncstream_dev.syncstream.model.ChatStream;
+import com.syncstream_dev.syncstream.model.Genre;
+import com.syncstream_dev.syncstream.model.VideoInfo;
+
+public class PrivateSession extends Session {
+    private String password;
+    private int maxViewers;
```

Comment:
If we are sticking to non-primitive types, then use `Integer`.

Reply:
We agreed to use primitive types, `Integer` -> `int` (which I already have) and `Long` -> `long`

Reply:
a2e382bdf3c2cdb06d1b89eef09f181dcc6e6ac9 addresses this